src/main/scala/org/hammerlab/guacamole/commands/jointcaller/VCFOutput.scala

```diff
@@ -79,7 +79,9 @@ object VCFOutput {
     extraHeaderMetadata.foreach(kv => {
       header.addMetaDataLine(new VCFHeaderLine(kv._1, kv._2))
     })
-    header.addMetaDataLine(new VCFHeaderLine("reference", reference.source))
+    if (reference.source.isDefined) {
```

Comment:
Yeah, totally subjective but I think as-is is actually clearer, so I think I'll leave it